### DISCUSSION TITLE
Welcome to autobahn-python Discussions!

### DISCUSSION THREAD

**Discussion by oberstet** [crossbario/autobahn-python#1623] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gggg990** [comment]
I cannot access crossbar.io, did the company close?

**Comment by nvxluca** [comment]
Hi, I've seen that the latest version is not compatible anymore with the pyInstaller library.
Is this topic something inside your roadmap?
Thank you.

**Reply by om26er** [reply]
it would make sense to open a new discussion topic. This one is just the announcement of the fact that this project will use Github for discussions. Also might make sense to actually open an issue instead

**Reply by nvxluca** [reply]
I've just open an issue, thanks.
